Firmware/planner.cpp

```diff
@@ -718,6 +718,8 @@ void plan_buffer_line(float x, float y, float z, const float &e, float feed_rate
 #endif /* PLANNER_DIAGNOSTICS */
   if(planner_aborted) {
       // avoid planning the block early if aborted
+      SERIAL_ECHO_START;
+      SERIAL_ECHOLNRPGM(_n("Planner aborted"));
```

Comment:
I find the message a bit confusing. I would expect this message when the planner is discarded, not when new moves are dropped due to a plan abort event. I would rename this to "Move aborted".

Reply:
I agree, it's done.